src/cpu/cpu.cpp

```diff
@@ -3097,13 +3098,6 @@ class Cpu final {
 	{
 		Section_prop* secprop = static_cast<Section_prop*>(sec);
 
-		{
-			PropMultiVal *p = secprop->GetMultiVal("cycles");
-			std::string type = p->GetSection()->Get_string("type");
-			std::string str;
-			CommandLine cmd("", p->GetSection()->Get_string("parameters"));
-		}
-
```

Comment:
Ok, I guess this entire block was removed in a later commit so you can ignore my unused variable comment.  Not entirely sure what you were doing here.

Reply:
Just some trace logging thing I was doing, but it's harmless so I can't be bothered to properly remove it from the commits...